### DISCUSSION TITLE
When will C# 12 be supported in CoPilot?

### DISCUSSION THREAD

**Discussion by yannickthibault** [community/community#116647] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Currently CoPilot supprot C# version 9.0, will newest versions of the C# language (like 10, 11, 12 and 13) be supported in Visual Studio and Visual Studio Code eventually and if yes any idea when?

**Comment by ayim** [comment, marked as answer]
Hey! @mikadumont are the best person to represent .NET.
I work on general language response quality ... and supporting newest conventions is definitely something we're looking into.

--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio
